login.php

```diff
@@ -46,8 +49,16 @@
     header("Location: {$URL}");
     exit;
 }
-//VALIDATE LOGIN INFORMATION
 else {
+    // Custom login loader
+    if (!$session->exists('login_custom.php')) {
+        require_once './login_custom.php';
+    }
+
+    if ($session->has('login_custom.php')) {
+        $page->write($session->get('login_custom.php'));
```

Comment:
Also here, as there isn't necessarily anything being cached or written to the page, right?